src/cli.js

```diff
@@ -73,39 +74,98 @@ async function Announce(model) {
+//             angel.mortal = mortal.uuid
+//             mortal.angel = angel.uuid
+//         }
+//     })
+// }
+
+// loadpaired data.txt FROM EXCEL
+async function loadPaired(workbook, model) {
+    worksheet = workbook.worksheets[0]
+    worksheet.eachRow(function(row, rowNumber) {
+        // Row 1 is the headers
+        if (rowNumber == 1) {
+            return
         }
-        if (name !== "") {
-            const newPerson = new Person().withName(name)
-            model.addPerson(newPerson)
-            console.log(newPerson.name + " - " + newPerson.uuid);
+        const angelName = row.getCell('A').value
+        if (model.getPersonByName(angelName)) {
+            console.error("Error: The angel's name " + angelName + " is already used!");
+            return;
         }
-    })
-    //TODO: print new uuids
-
-    content.split("\n").forEach(line => {
-        if (line.trim() === "") {
+        if (angelName == "") {
+            console.error("Error: The name at rowNumber " + rowNumber + " is empty!");
             return;
         }
-        const angelName = line.split(",")[0].trim()
-        const mortalName = line.split(",")[1].trim()
-        if (angelName === "" || mortalName === "") {
-            console.error("Invalid line: " + line)
-        } else {
-            console.log(`${angelName}-${mortalName}`)
-            const angel = model.getPersonByName(angelName)
-            const mortal = model.getPersonByName(mortalName)
-            // console.log(a, m)
-            angel.mortal = mortal.uuid
-            mortal.angel = angel.uuid
+        const newPerson = new Person().withName(angelName)
+        model.addPerson(newPerson)
+        console.log(newPerson.name + " - " + newPerson.uuid);
+    });
+
+    worksheet.eachRow(function(row, rowNumber) {
+        if (rowNumber == 1) {
+            return
         }
-    })
+        const angelName = row.getCell('A').value
+        const angel = model.getPersonByName(angelName)
+        row.eachCell(function(cell, colNumber) {
+            // console.log('Cell ' + colNumber + ' = ' + cell.value)
+            // First column contains angelName
+            if (colNumber == 1) {
+                return
+            }
+            // Cell contains mortal name
+            if (colNumber == 2) {
+                const mortalName = cell.value;
+                if (mortalName === "") {
+                    console.error(`Invalid mortal name at row ${rowNumber}, col ${colNumber}`);
+                    return;
+                }
+                console.log(`${angelName}-${mortalName}`)
+                const mortal = model.getPersonByName(mortalName)
+                // console.log(a, m)
+                angel.mortal = mortal.uuid
+                mortal.angel = angel.uuid
+                return;
+            }
+            // Cell contains fun fact about the person
+            if (colNumber > 2 && colNumber < 8) {
+                angel.facts.push(cell.value)
```

Comment:
this implicitly assumes the eachCell iteration order. It would be better defensive programming to construct the person with a fixed size array, then index that array based on colNumber